lib/effects/sequence_effect.dart

```diff
@@ -36,9 +36,6 @@ class SequenceEffect extends PositionComponentEffect {
   @override
   void initialize(PositionComponent component) {
     super.initialize(component);
```

Comment:
IIRC @spydon had a reason why this needed to be on initialize and not on the constructor/inits. not sure now what it was, just flagging for context

Reply:
I would be happy to revert this adding an appropriate comment to the code.

Reply:
@spydon could you please elaborate on this? It's about `_currentIndex` and `_driftModifier` defaults in `SequenceEffect`. Thanks!

Reply:
It needs to be there since `reset` is using `initialize` if I remember correctly.

Reply:
> It needs to be there since `reset` is using `initialize` if I remember correctly.

I've added `_initialIndex` and `_initialDriftModifier` constants. IMO, this way the code looks more clear in its intent now.

Reply:
I would rather see this as a comment, this feels a bit over-engineered, especially since those values aren't used anywhere else. If I didn't write this code and I got here I would have to look up what the initial values were, instead of seeing them directly.